RubeesEat.IntegrationTests/PageObjects/UserHomepagePageObject.cs

```diff
@@ -14,13 +14,13 @@ public string[] GetBalanceChangeTexts()
 
     public BalanceChangeControlObject[] GetBalanceChanges()
     {
-        return FindElements(By.CssSelector(".entryLines > li"))
+        return FindElements("entryLines > li")
```

Comment:
This does not look right. Why is there a CSS selector?

Reply:
What do you mean exactly? We need the selector to get the entryLines displayed as a list.

Reply:
I don't get why we can call `FindElements` with a css selector. In the current task we are supposed to clean up finding element by id so this change does not seem relevant.

Reply:
The line is a bit confusing: FindElements looks for a data-test-id, but in this case I set the data-test-id to "entryLines > li" (so that it is clear from the code, that it is looking for lis in entryline). I can change the data-test-id to something like "entrylinesLine" or something like that, if that is better.

But since you said we only want to clean up finding by id, we can revert this line to look for CssSelector, as before

Reply:
It should not look like a CSS selector if it is not a CSS selector. We can use a singular name `entryLine` or `balanceChange` as ID for each element and search for that instead.